### DISCUSSION TITLE
What is the shortest time series on which TSFEL could be computed?

### DISCUSSION THREAD

**Discussion by irazza** [fraunhoferportugal/tsfel#137] [Q&A]
I am interested in understanding whether there exists a minimum length requirement or a lower bound for the length of a time series.

**Comment by dmfolgado** [comment]
Hi! Thank you for your interest in tsfel. We did not enforce a minimum length requirement for the time series as it depends on the use case. However, please be aware that some features might not work with a minimum number of samples, e.g., some temporal and spectral features.
